### PR TITLE
CP-17066: Pass monitor config file to xenopsd for GVT-g equipped VMs

### PR DESCRIPTION
**Pull request by johnelse**

### CODE REVIEW

**Comment by johnelse** [comment]
Depends on https://github.com/xapi-project/xcp-idl/pull/121

File: ocaml/xapi/xapi_vgpu_type.ml [hunk 1]
```diff
@@ -487,7 +487,12 @@ module Intel = struct
 						Xapi_globs.vgt_low_gm_sz, Int64.to_string conf.identifier.low_gm_sz;
 						Xapi_globs.vgt_high_gm_sz, Int64.to_string conf.identifier.high_gm_sz;
 						Xapi_globs.vgt_fence_sz, Int64.to_string conf.identifier.fence_sz;
-					];
+					] @ (
+						match conf.identifier.monitor_config_file with
```
Comment (robhoes) [inline ocaml/xapi/xapi_vgpu_type.ml]: Convenient that `monitor_config_file` was already in `type gvt_g_id`.

**Comment by robhoes** [comment]
Looks good. :shipit:

File: ocaml/xapi/xapi_vgpu_type.ml (hunk 1 above)
Comment (johnelse) [inline ocaml/xapi/xapi_vgpu_type.ml]: Yep - it was already being read from the master GVT-g config file, but wasn't being sent to xenopsd.

**Pull request merged**